### PR TITLE
Indonesia map

### PR DESCRIPTION
**Pull request by muhammad-ihsan**
adding indonesia map

### CODE REVIEW

File: packages/indonesia/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Sulawesi Selatan
+* Sulawesi Barat
+* Sumatera Selatan
+* Sulawesi Tengah
+* Sumatera Utara
+* Yogyakarta
+* Timor-Leste
+* Sabah
+* Sarawak
+* Brunei Darussalam
```
[the remark this answers is not in the document]
Comment (muhammad-ihsan) [reply packages/indonesia/README.md:41]: yes totally it is 🇲🇾  i only following the svg source. I will fix it and update the svg too

**Comment by VictorCazanave** [comment]
Hi @muhammad-ihsan ,
If you still want to add Indonesia map, could you push the modifications of #28 to this PR please?

**Comment by muhammad-ihsan** [comment]
yes please check the update @VictorCazanave

File: packages/indonesia/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# 🇮🇩 Indonesia
+
+Contains all the states and union territories of Indonesia:
```
[the remark this answers is not in the document]
Comment (VictorCazanave) [reply packages/indonesia/README.md:3]: Could you fix the name or provide a better source please?

**Comment by VictorCazanave** [comment]
Thanks @muhammad-ihsan ! Could you fix the remaining comments and resolve the conflicts please?

File: packages/indonesia/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (muhammad-ihsan) [reply packages/indonesia/README.md]: already update, please re-check mate @VictorCazanave

File: packages/indonesia/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+The original map is available [here](https://mapsvg.com/maps/indonesia) and free to be used in any custom project.
+
+### Changes
+
+* Remove unnecessary attributes
+* Replace title by name attributes
+* Add viewBox
+* Rename ids
+* Sort `<path/>` alphabetically
```
[the remark this answers is not in the document]
Comment (muhammad-ihsan) [reply packages/indonesia/README.md:55]: please re0check it mate @VictorCazanave

File: packages/indonesia/indonesia.svg [hunk 4]
```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
[the remark this answers is not in the document]
Comment (VictorCazanave) [reply packages/indonesia/indonesia.svg]: Sorry but the SVG doesn't have the right format. You can look at the other maps or follow the updated [contribution guide](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#svg-file).

**Pull request closed without merging**